src/sbt-test/sbt-bundle/basic/build.sbt

```diff
@@ -15,6 +15,22 @@ BundleKeys.roles := Set("web-server")
 
 BundleKeys.endpoints += "other" -> Endpoint("http", 0, Set(URI("http://:9001/simple-test")))
 BundleKeys.endpoints += "akka-remote" -> Endpoint("tcp")
+BundleKeys.endpoints += "extras" -> Endpoint("http", 0, "ping-service",
+  Acl(
+    HttpRequestMappings(
+      HttpPath("/bar-1"),
+      HttpPath("/bar-2", "GET"),
+      HttpPath("/bar-3", "GET", "/other-bar-3"),
+      HttpPathBeg("/beg-1"),
+      HttpPathBeg("/beg-2", "GET"),
+      HttpPathBeg("/beg-3", "GET", "/other-beg-3"),
+      HttpPathRegex("/regex1/[a|b|c]"),
+      HttpPathRegex("/regex2/[a|b|c]", "GET"),
+      HttpPathRegex("/regex3/[a|b|c]", "GET", "/other-regex-3")
+    )),
+  Acl(TcpRequestMappings(9001, 9002)),
+  Acl(UdpRequestMappings(19001, 19002))
+)
```

Comment:
@markusjura Your thoughts too please. I realise that Lagom will alleviate the developer having to specify the above because it will be generated. However I'm primarily considering the non-Lagom use-cases and wanting build.sbt to look nice.